src/libraries/Common/src/System/Security/Cryptography/SlhDsa.cs

```diff
@@ -1728,29 +1728,29 @@ public static SlhDsa ImportSlhDsaPublicKey(SlhDsaAlgorithm algorithm, byte[] sou
         ///   The platform does not support SLH-DSA. Callers can use the <see cref="IsSupported" /> property
         ///   to determine if the platform supports SLH-DSA.
         /// </exception>
-        public static SlhDsa ImportSlhDsaSecretKey(SlhDsaAlgorithm algorithm, ReadOnlySpan<byte> source)
+        public static SlhDsa ImportSlhDsaPrivateKey(SlhDsaAlgorithm algorithm, ReadOnlySpan<byte> source)
         {
             ArgumentNullException.ThrowIfNull(algorithm);
 
-            if (source.Length != algorithm.SecretKeySizeInBytes)
+            if (source.Length != algorithm.PrivateKeySizeInBytes)
             {
-                throw new ArgumentException(SR.Argument_SecretKeyWrongSizeForAlgorithm, nameof(source));
+                throw new ArgumentException(SR.Argument_PrivateKeyWrongSizeForAlgorithm, nameof(source));
```

Comment:
@vcsjones I think this is the line you were looking for.

Reply:
@bartonjs what about Cryptography_MLDsaNoSecretKey? I don’t see that added or removed anywhere in the PR.